jobs/cloud_controller_ng/spec

```diff
@@ -395,7 +395,7 @@ properties:
 
   cc.run_prestart_migrations:
     description: "Run Cloud Controller DB migrations in BOSH pre-start script. Should be changed to false for deployments where the PostgreSQL job is deployed to the same VM as Cloud Controller. Otherwise, the default of true is preferable."
-    default: true
+    default: false
```

Comment:
In case we want to change the default value, the description should be adapted as well.

Reply:
yes, exactly. thank you